space2stats_api/src/space2stats/__main__.py

```diff
@@ -0,0 +1,23 @@
+
+import os
+
+from .app import app
+
+try:
+    import uvicorn  # noqa
```

Comment:
do we need a try block here? Couldn't we just let it fail hard if dependencies aren't installed?

Reply:
if we don't have the try/except here and just have `import Uvicorn` then the whole package might fail.

having the try/except here ensure that the code will always work